### PR TITLE
Exchange lat/long, flip vertically to match Swing app

### PR DESCRIPTION
**Pull request by pilhuhn**
Update the export code so that the orientation of the generated SVG matches what is visible in the Swing app.

Bildschirmfoto 2021-01-18 um 21 13 03

### CODE REVIEW

**Comment by triceo** [comment]
OK to test.

File: optaplanner-examples/src/main/java/org/optaplanner/examples/tsp/persistence/SvgTspPathExporter.java [hunk 1]
```diff
@@ -54,21 +54,25 @@ public void writeSolution() throws IOException {
...
+                bufferedWriter.write((location.getLongitude() + offsetX) + ",");
+                bufferedWriter.write((height - (location.getLatitude() + offsetY)) + "\n");
                 standstill = findNextVisit(standstill);
             }
             // Now return home
             bufferedWriter.write("L ");
-            bufferedWriter.write((home.getLatitude() + offsetX) + ",");
-            bufferedWriter.write((home.getLongitude() + offsetY) + "\n");
+            bufferedWriter.write((home.getLongitude() + offsetX) + ",");
+            bufferedWriter.write((height - (home.getLatitude() + offsetY)) + "\n");
```
Comment (triceo) [inline optaplanner-examples/src/main/java/org/optaplanner/examples/tsp/persistence/SvgTspPathExporter.java:75]: I'm thinking that this code would be better readable if these calculations were extracted into methods with descriptive names. It'd also help with the considerable repetition here.

**Review by triceo — APPROVED** [review]
LGTM. Added a suggestion to improve readability of the code.

**Comment by ge0ffrey** [comment]
@triceo Let's merge and do that on our end?

**Pull request merged**